scripts/env/ci.sh

```
@@ -0,0 +1,266 @@
+    ./node_modules/.bin/next dev &
+    DEV_PID=$!
+    
+    # Wait for server to be ready
+    log_info "Waiting for dev server to be ready"
+    for i in {1..60}; do
+        if curl -f http://localhost:3000 >/dev/null 2>&1; then
+            log_success "Dev server is ready"
+            break
+        fi
+        sleep 3
+        if [ $i -eq 60 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)